hack/dockerized

```diff
@@ -3,6 +3,11 @@ set -e
 
 source $(dirname "$0")/common.sh
 
+if [ "${KUBEVIRT_RUN_UNNESTED}" == "true" ]; then
```

Comment:
I tried this, for me it didn't work. What I did was:

```
> export KUBEVIRT_RUN_UNNESTED=true
> make generate
hack/dockerized "DOCKER_PREFIX= DOCKER_TAG= IMAGE_PULL_POLICY= VERBOSITY= ./hack/generate.sh"
hack/dockerized: line 7: DOCKER_PREFIX= DOCKER_TAG= IMAGE_PULL_POLICY= VERBOSITY= ./hack/generate.sh: No such file or directory
make: *** [Makefile:29: generate] Error 127
```

Reply:
Thanks. It seems like within our builder container, the `$@` is executed with int `sh` mode. Changed it to `/bin/bash -c "$@"` and it started to work.

This works now:

```bash
hack/dockerized /bin/bash
export KUBEVIRT_RUN_UNNESTED=true
make generate
```

Reply:
For me this worked:

```
hack/dockerized "export KUBEVIRT_RUN_UNNESTED=true && make generate"
```

I think we should document this somehow as the usual approach people would try to use this would be the above one. And then they'll start to complain, no?